src/expr/impl/src/scalar/trigonometric.rs

```diff
@@ -437,6 +440,32 @@ pub fn acosd_f64(input: F64) -> F64 {
     result.into()
 }
 
+// return the invert tangent of x in degrees, the inverse tangent function maps all inputs to
+// values in the range [-90, 90]. For the 5 special case inputs (0, 1, -1, +INF and -INF), this
+// function will return exact values (0, 45, -45, 90 and -90 degrees respectively).
+#[function("atand(float8) -> float8")]
+pub fn atand_f64(input: F64) -> F64 {
+    if input.0.is_nan() {
+        return F64::from(f64::NAN);
+    }
+
+    if input.0.is_infinite() {
+        if input.0.is_sign_positive() {
+            return DEGREE_NINETY.into();
+        } else {
+            return (-DEGREE_NINETY).into();
+        }
+    }
+
+    let atan_arg1 = atan_f64(input);
+    let result = (atan_arg1 / atan_f64(DEGREE_ONE.into())) * DEGREE_FORTY_FIVE;
+
+    if result.0.is_infinite() {
+        return F64::from(f64::NAN);
+    }
+    result
```

Comment:
```suggestion
    let arg1 = input.0;

    if arg1.is_nan() {
        return F64::from(f64::NAN);
    }

    let atan_arg1 = f64::atan(arg1);
    let result = (atan_arg1 / f64::atan(DEGREE_ONE)) * 45.0;

    if result.is_infinite() {
        return F64::from(f64::NAN);
    }
    result.into()
```

Reply:
1. `f64::atan(inf)` and `f64::atan(-inf)` can be calculated correctly, no need to specially handled.
2. The `DEGREE_ONE` used in `atan` to calculate `atan_1_0` is for avoiding compile time optimization. We should not use `DEGREE_FORTY_FIVE` and `DEGREE_NINETY` outside `atan` function call.